book/content/version_control/vcs-github.md

```diff
@@ -0,0 +1,184 @@
+## Using Git commands to work on GitHub
+
+As research becomes increasingly collaborative and multiple people work on the same project, it becomes difficult to keep track of changes made by others if not done systematically.
+Moreover, it is time-consuming to manually incorporate the work of different participants in a project, even when all of their changes are compatible.
+Hosting the project on an online distributed version control system like GitHub is beneficial to make collaborations open and effective.
```

Comment:
Sorry for this nitpick. Git is a distributed version control system, GitHub is a web-based hosting service for Git repositories. Its hard to disentangle the two some times, but it is so helpful to clearly distinguish them - else people may try the GitHub documentation for problems they are having with Git commands and should consult Git's documentation for.

```suggestion
Hosting the project on an online repository hosting service like GitHub is beneficial to make collaborations open and effective.
```

Reply:
Absolutely agree on that, and I appreciate you pointing these out.